lib/dotcom/system_status/subway.ex

```diff
@@ -109,6 +119,7 @@ defmodule Dotcom.SystemStatus.Subway do
         "Green" => [%{branch_ids: [], status_entries: [%{time: :current, status: :normal, multiple: false}]}]
       }
   """
+  @spec subway_status([Alert.t()], DateTime.t()) :: %{String.t() => status_entry_group()}
```

Comment:
Curious if the return type can also be described as `%{Routes.Route.id_t() => status_entry_group()}` or even literally
```
%{
  "Blue" => status_entry_group(),
  "Green" => status_entry_group(),
  "Orange" => status_entry_group(),
  "Red" => status_entry_group()
}
```